web/styles/subscriptions.css

```diff
@@ -1392,6 +1392,30 @@ div.settings-radio-input-parent {
     margin: -6px 0 -6px -10px;
 }
 
+#deactivation-confirm-modal {
+    .alert {
+        padding-right: 14px;
```

Comment:
Probably we should write new sizes in `em` unless there's a good reason not to. Not worth changing for this PR, but FYI.

Reply:
We currently use `px` for paddings at various places where properties like width, height use `em`, so used that. But yeah using `em` would be better. Will change in a follow-up PR.